site/bin/_lib.sh

```diff
@@ -314,8 +324,11 @@ release() {
   echo "     ... replace title in versioned mkdocs.yml"
   sed -i'' -E "s/(^site_name:[[:space:]]+).*$/\1\"Nessie ${RELEASE_VERSION}\"/" "${target}/mkdocs.yml"
 
-  echo "     ... replace version placeholder in versioned docs"
-  find "${target}" -name "*.md" -exec sed -i "s/::NESSIE_VERSION::/${RELEASE_VERSION}/g" {} \;
+  echo "     ... replace version placeholders in versioned docs"
+  find "${target}" -name "*.md" -exec sed -i='' "s/::NESSIE_VERSION::/${RELEASE_VERSION}/g" {} \;
```

Comment:
According to this [SO question](https://stackoverflow.com/questions/12272065/sed-undefined-label-on-macos), the syntax `-i=''` should be portable across different OSes (just `-i` gives an error on macOS).

Reply:
Seems to work, yes